### PR TITLE
merge (RAW)KV_COMMAND_COUNTER into COMMAND_COUNTER

### PR DESCRIPTION
**Pull request by breeswish**
Previously, we have two command counters in storage: KV_COMMAND_COUNTER for KV commands and RAWKV_COMMAND_COUNTER for raw KV commands.

However, in grafana dashboard, we only displays KV_COMMAND_COUNTER and name it as "storage command total", which is very misleading and is lack of raw metrics. In addition, we use cmds in KV_COMMAND_COUNTER to repeatedly set up panels, so that we don't have panels like "Scheduler - Raw Get".

Due to these reasons, this PR merge KV_COMMAND_COUNTER and RAWKV_COMMAND_COUNTER into COMMAND_COUNTER. The metric name is not changed, still "tikv_storage_command_total". It will solve problems above.

@siddontang @zhangjinpeng1987 @hicqu PTAL

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: src/storage/metrics.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@
 use prometheus::*;
 
 lazy_static! {
-    pub static ref KV_COMMAND_COUNTER_VEC: IntCounterVec =
+    pub static ref COMMAND_COUNTER_VEC: IntCounterVec =
```
Comment (huachaohuang) [inline src/storage/metrics.rs]: I think we can still name it `KV_COMMAND_COUNTER_VEC`, just merge `RAWKV_COMMAND_COUNTER_VEC` into it.

**Comment by breeswish** [comment]
@siddontang Oh, my fault, forgot to make a commit. Updated. PTAL

**Comment by siddontang** [comment]
PTAL @huachaohuang

**Pull request merged**